install_config/certificate_customization.adoc

```diff
@@ -137,9 +137,9 @@ openshift_master_named_certificates=[{"certfile": "/path/to/console.ocp-c1.myorg
 
 Where the parameter values are: 
 
-* *certfile* is the path to the file that contains the {product-title} router certificate.
+* *certfile* is the path to the file that contains the {product-title} cluster certificate.
 
-* *keyfile* is the path to the file that contains the {product-title} wildcard key.
+* *keyfile* is the path to the file that contains the {product-title} cluster key.
```

Comment:
@mrogers950, is this a good way to describe the master certificate and key?

Reply:
I'd suggest saying `named` or `custom` certificate/key here, since `cluster` could elsewhere refer to internal cluster certificates that are not trusted by external users.

Reply:
+1 to `custom`.

@bparry02, if you make the update, I'll merge. If not, I'll open a new PR to fix this the next time I swing through PRs.

Reply:
(If you make the change, please squash your commits.)

Reply:
@kalexand-rh I updated the description to `custom`